app/screens/TalkDetailsScreen/BreakDetailsScreen.tsx

```diff
@@ -0,0 +1,229 @@
+  subtitle: string
+  /**
+   * The banner image
+   */
+  imageUrl: RecurringEvents["secondary-callout-banner"]
+  /**
+   * The Break Sponsor
+   */
+  sponsor: Sponsor
+  /**
+   * The description of the break
+   */
+  description: string
+  /**
+   * The location of the break
+   */
+  location: string
+  /**
+   * The calloutLink of the break
+   */
+  calloutLink: string
+}
+
+const breakDetailsProps = (schedule: ScheduledEvent, sponsors: Sponsor[]): BreakDetailsProps => {
+  const recurringEvent = schedule["recurring-event"]
+  const sponsor = sponsors.find(
+    (s) => s._id === recurringEvent["sponsor-for-secondary-callout-optional"],
+  )
+  return {
+    title: recurringEvent["secondary-callout"],
+    subtitle: `${formatDate(schedule["day-time"], "MMMM dd, h:mmaaa")} PT`,
+    imageUrl: recurringEvent["secondary-callout-banner"],
+    sponsor,
+    description: recurringEvent["secondary-callout-description"],
+    location: recurringEvent["secondary-callout-location"],
+    calloutLink: recurringEvent["secondary-callout-link"],
+  }
+}
+
+export const BreakDetailsScreen: FC<StackScreenProps<AppStackParamList, "BreakDetails">> = ({
+  route: { params },
+}) => {
+  const [headingHeight, setHeadingHeight] = React.useState(0)
+
+  const { scrollHandlers } = useFloatingActionEvents()
+  const { scrollY, scrollHandler } = useScrollY()
+  const { bottom: paddingBottom } = useSafeAreaInsets()
+
+  const { data: scheduleData } = useScheduledEventsData()
+  const { data: sponsors = [] } = useSponsors()
+  const schedule = scheduleData?.find((s) => s._id === params?.scheduleId)
+
+  if (!schedule) return null
+
+  const { title, subtitle, imageUrl, sponsor, description, location, calloutLink } =
+    breakDetailsProps(schedule, sponsors)
+
+  return (
+    <Screen safeAreaEdges={["top", "bottom"]} style={$root}>
+      <TalkDetailsHeader {...{ title, subtitle, scrollY, headingHeight }} />
+
+      <Animated.ScrollView
+        style={[$scrollView, { paddingBottom }]}
+        scrollEventThrottle={16}
+        onScroll={scrollHandler}
+        showsVerticalScrollIndicator={false}
+        {...scrollHandlers}
+      >
+        <View style={$container}>
+          <View style={$headingContainer}>
+            <Text
+              preset="screenHeading"
+              style={$title}
+              text={title}
+              onLayout={({
+                nativeEvent: {
+                  layout: { height },
+                },
+              }) => {
+                setHeadingHeight(height)
+              }}
+            />
+            <Text preset="companionHeading" style={$subtitle} text={subtitle} />
+          </View>
+
+          <View style={$containerSpacing}>
+            <Image source={imageCurve} style={$imageCurve} />
+            <Image
+              source={
+                (imageUrl as ImageRef).url
+                  ? { uri: (imageUrl as ImageRef).url }
+                  : (imageUrl as ImageRequireSource)
+              }
```

Comment:
The consumer of `imageURL` has to know that it could be of type `ImageRequireSource` and account for it here. If I was looking at it just from the prop name, I wouldn't have guessed that. If pushed it to the data layer, we could call the prop `image` and the consumer could use it as `source={image}` without knowledge of what it was.